qutebrowser/browser/shared.py

```diff
@@ -421,8 +436,9 @@ def _execute_fileselect_command(
             message.error(f"Failed to open tempfile {tmpfilename} ({e})!")
             selected_files = []
 
-    if not multiple:
+    if not qb_mode == FileSelectionMode.MULTIPLE_FILES:
+        # TODO should we assert if a file(s)/folder was actually chosen?
```

Comment:
This is what I found:
* Providing a file path which doesn't exist gives the warning
   ```
   WARNING: Ignoring invalid item in FilePickerController::accepted(QStringList): non-existent
    ```
    and the file is correctly ignored. The same for single_file, multiple_files and folder.
* Selecting a directory path instead of a file or vice versa does get accepted in the test page. I also tried it here to select a file to attach to the message and instead choose a folder. This triggered the error from github:
  ```
   We don't support that file type. Try again with a GIF, JPEG, JPG, PNG, DOCX, GZ, LOG, PDF, PPTX, TXT, XLSX or ZIP
   ```
    (there were two .txt files in the folder)